paasta_tools/kubernetes_tools.py

```diff
@@ -1440,6 +1441,9 @@ def format_kubernetes_app(self) -> Union[V1Deployment, V1StatefulSet]:
     def get_kubernetes_service_account_name(self) -> Optional[str]:
         return self.config_dict.get("service_account_name", None)
 
+    def is_istio_sidecar_injection_enabled(self) -> bool:
+        return self.config_dict.get("is_istio_sidecar_injection_enabled")
```

Comment:
I think we want 
```suggestion
        return self.config_dict.get("is_istio_sidecar_injection_enabled", False)
```
otherwise we might get a `None` here